src/main/java/de/tum/in/www1/artemis/web/rest/UserResource.java

```diff
@@ -84,19 +87,35 @@ public UserResource(UserRepository userRepository, UserService userService, User
         this.authorityRepository = authorityRepository;
     }
 
+    private static void checkUsernameAndPasswordValidity(String username, String password) {
+
+        if (!StringUtils.hasLength(username) || username.length() < USERNAME_MIN_LENGTH) {
+            throw new AccessForbiddenException("The username has to be at least " + USERNAME_MIN_LENGTH + " characters long");
+        }
+
+        // Note: the password can be null, then a random one will be generated (Create) or it won't be changed (Update).
+        // If the password is not null, its length has to be at least PASSWORD_MIN_LENGTH
+        if (password != null && password.length() < PASSWORD_MIN_LENGTH) {
+            throw new AccessForbiddenException("The password has to be at least " + PASSWORD_MIN_LENGTH + " characters long");
+        }
+    }
+
     /**
      * POST /users : Creates a new user.
      * <p>
```

Comment:
```suggestion
     *
```
maybe we should start to remove all those weird `<p>` tags, they don't really make sense, as @FrankeLukas made me notice recently.